observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -8,10 +8,142 @@ toc::[]
+You can use the {TempoName} xref:../otel/otel-forwarding.adoc#otel-forwarding-traces[in combination with] the xref:../otel/otel-rn.adoc#otel_rn[{OTELName}].
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
+[id="distr-tracing_3-3_{context}"]
+== Release notes for {DTProductName} 3.3
+
+This release of the {DTProductName} includes the {TempoName} and the deprecated {JaegerName}.
+
+////
+[id="distr-tracing_3-3_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-202?-????/[CVE-202?-????]
+////
+
+[id="distr-tracing_3-3_tempo-release-notes_{context}"]
+=== {TempoName}
+
+The {TempoName} is provided through the {TempoOperator}.
+
+The {TempoName} 3.3 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.5.0.
+
+////
+[id="distr-tracing_3-3_tempo-release-notes_technology-preview-features_{context}"]
+==== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* ???.
+
+:FeatureName: The Tempo monolithic deployment
+include::snippets/technology-preview.adoc[leveloffset=+1]
+////
+
+[id="distr-tracing_3-3_tempo-release-notes_new-features-and-enhancements_{context}"]
+==== New features and enhancements
+
+This update introduces the following enhancements:
+
+* Added support for securing Jaeger UI and Jaeger APIs with OpenShift Oauth proxy. (link:https://issues.redhat.com/browse/TRACING-4108[TRACING-4108])
+* Added support for using OpenShift serving certificates on ingestion APIs when multitenancy is disabled. (link:https://issues.redhat.com/browse/TRACING-3954[TRACING-3954])
+* Added support for ingesting by using the OTLP/HTTP protocol when multitenancy is enabled. (link:https://issues.redhat.com/browse/TRACING-4171[TRACING-4171])
+* Added support for AWS S3 Secure Token authentication. (link:https://issues.redhat.com/browse/TRACING-4176[TRACING-4176])
+* Added support for automatically reloading certificates. (link:https://issues.redhat.com/browse/TRACING-4185[TRACING-4185])
+* Added support for configuring the duration for which service names are available for query. (link:https://issues.redhat.com/browse/TRACING-4214[TRACING-4214])
+
+////
+[id="distr-tracing_3-3_tempo-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+In the {TempoName} 3.3, ???.
+////
+
+////
+[id="distr-tracing_3-3_tempo-release-notes_removal-notice_{context}"]
+==== Removal notice
+
+In the {TempoName} 3.3, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+////
+
+[id="distr-tracing_3-3_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fixes:
```

Comment:
https://issues.redhat.com/browse/TRACING-4511 should be added as bug fix.

> Before this update Jaeger UI could fail on `504 Gateway Time-out` when it was accessed via route. After this update, users can specify OpenShift route annotations for increasing timeout (`haproxy.router.openshift.io/timeout: 3m`) when querying large data sets (link:).

Reply:
Thank you, Pavol. I've added it to the file.